tests_integration/conftest.py

```diff
@@ -27,23 +27,22 @@
 
 import pytest
 import yaml
+from typer import Typer
+from typer.testing import CliRunner
 
+from snowflake.cli._app.cli_app import app_factory
 from snowflake.cli.api.cli_global_context import (
     fork_cli_context,
     get_cli_context_manager,
 )
 from snowflake.cli.api.connections import OpenConnectionCache
-from snowflake.cli._app.cli_app import app_factory
-from typer import Typer
-from typer.testing import CliRunner
-
 from snowflake.cli.api.project.util import TEST_RESOURCE_SUFFIX_VAR
 from tests.conftest import clean_logging_handlers_fixture  # noqa: F401
+from tests.testing_utils.files_and_dirs import merge_left
 from tests.testing_utils.fixtures import (
-    alter_snowflake_yml,  # noqa: F401
+    # noqa: F401
```

Comment:
nit: F401 is "unused import", but it's been removed so the `noqa` can be removed too

Reply:
I'm not entirely sure why that import got removed, I'm guessing my IDE or pre-commit did it automatically, but all the tests are passing so I guess I'll remove the comment as well and hope for the best 👍